src/util/variant.ts

```diff
@@ -2,4 +2,5 @@ export enum VariantKey {
   Default = "version_default",
   Dotnet = "version_dotnet",
   Docker = "version_docker",
+  Major = "version_major",
```

Comment:
I wish these were simpler, what do you think about adding variants without a preceding `version_` which feels more and more redundant:

```

Also what do you think about having them with simplified names:

```ts
export enum VariantKey {
  Default = "default",
  Dotnet = "dotnet",
  Docker = "docker",
  Major = "major",
}
```